lesson06/assignment/scripts.js

```diff
@@ -0,0 +1,19 @@
+
+function validateForm() {
+  const userName = document.forms["contactForm"]["username"].value;
+  const email = document.forms["contactForm"]["email"].value;
+  const messageTextarea = document.forms["contactForm"]["messageTextarea"].value;
+}
+
+  if (userName == "") {
+    alert("Please enter your name.");
+    return false;
+  } else if (email == "") {
+    alert("Please enter your email.");
+    return false;
+  } else if (messageTextarea.value == "") {
+    alert("Please enter a message.");
+    return false;
+  }
+  else alert("Thank you, we'll contact you soon!");
+}
```

Comment:
Two problems: 
- the closing bracket on line 19 doesn't have a partner, so will prevent the JS in the file from compiling
- this validation logic isn't tied to the form in any way
Review lesson 01 for a refresher on DOM interactions.